### ISSUE TITLE
Can't locate Template.pm in @INC

### ISSUE THREAD

**Issue by eserte** [xsawyerx/template-plugin-cgi#1]
t/cgi.t fails if Template.pm is not yet installed:
```
Can't locate Template.pm in @INC (you may need to install the Template module) (@INC contains: ...  .) at t/cgi.t line 21.
BEGIN failed--compilation aborted at t/cgi.t line 21.
t/cgi.t ................ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```
Probably the dependency needs to be specified.

**Comment by toddr** [comment]
LOL beat me to it! Thank you

**Comment by toddr** [comment]
fixed in a8d2bacc7156f9aa601ca8ec4e77e8818eb2bd0b. Should be in next release today.

**Comment by toddr** [comment]
This is now on CPAN as 3.101
